index.bs

```diff
@@ -584,7 +632,12 @@ and an {{IntersectionObserverInit}} dictionary |options|, run these steps:
 8. If |thresholds| is empty, append <code>0</code> to |thresholds|.
 9. The {{IntersectionObserver/thresholds}} attribute getter will return
 	this sorted |thresholds| list.
-10. Return |this|.
+10. If |options|.{{IntersectionObserverInit/trackVisibility}} is <code>true</code>,
+	and |options|.{{IntersectionObserverInit/delay}} is not specified or is less than <code>100</code>,
+	<a>throw</a> a {{NotAllowedError}} exception.
```

Comment:
My thought was that it would be more helpful to developers to confront them with a console error when `delay` is too small, rather than quietly clamp the value. What do you think about clamping the value to 100ms and *also* printing a console warning?

Reply:
"tough love" in the form of throwing for a too-small value is certainly something you can do. But it complicates standardizing this by raising the stakes: everyone has to be on board with the specific minimum value that's going to be in the spec,  once set no browser could ever decide to use a higher minimum (pages all over would break in that browser), and if a browser decides to support a lower minimum, every web site that decides to take advantage of that minimum will break in all the other browsers.

As an implementation, a console warning for a clamped value is a good idea, but I'm not sure console warnings are supposed to be part of a spec like this. What if a user agent doesn't have a console? How does a console warning get covered by web platform tests? A non-normative suggestion might fly though.

Reply:
OK, I'm fine with automatically clamping to the UA's minimum, and I'll probably add a console warning to chromium.